python/tvm/relay/frontend/tflite.py

```diff
@@ -1770,6 +1775,12 @@ def convert_pool2d(self, op, pool_type):
                 assert self.has_same_qnn_params(input_tensor, output_tensor), \
                         "qnn.op.max_pool2d requires input and output qnn params to be same"
             out = _op.nn.max_pool2d(in_expr, **params)
+        elif pool_type == "l2":
+            # l2_pool_2d is equivalent to sqrt(avg_pool(sqr(in_data)))
```

Comment:
Better name `sqrt` to `square_root`, `sqr` to `square`. `sqrt` is too similar with `sqr` so that code readers confuse easily.

Reply:
corrected